Programming_in_Scala/Chap4_Classes_Objects/Readme.md

```diff
@@ -0,0 +1,32 @@
+
+
+->	In scala primitive data types are also considered as Objects.
+
+->	Scala Int object is converted internally in java.int or Integer Objects.
+
+->	Fuctions inside a class is know as methords and when it is inside object its 	 know as functions.
+
+-> methords without "=" will always return Unit.
+
+-> Main function dont have "=" as its expected to return Unit.
+
+-> Methords that dont return a value is called procedure.
```

Comment:
Add on:- methods without "=" returns unit and these methods are called as procedures